### PR TITLE
app-text/atril: add 1.26.2

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

File: app-text/atril/atril-1.26.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		<dev-libs/mathjax-3
+		>=net-libs/webkit-gtk-2.6.0:4.1
+	)
+	keyring? ( >=app-crypt/libsecret-0.5 )
+	introspection? ( >=dev-libs/gobject-introspection-0.6:= )
+	postscript? ( >=app-text/libspectre-0.2 )
+	synctex? ( virtual/tex-base )
+	tiff? ( >=media-libs/tiff-3.6:= )
+	xps? ( >=app-text/libgxps-0.2.1 )
+"
```
Comment (juippis) [inline app-text/atril/atril-1.26.2.ebuild:53]: You're missing `app-arch/libarchive:=`, per 
```
+COMICS_DEPS="libarchive"
 if test "x$enable_comics" = "xyes"; then
     AC_DEFINE([ENABLE_COMICS], [1], [Enable support for comics.])
+    PKG_CHECK_MODULES([COMICS], [$COMICS_DEPS])
+    AC_SUBST(COMICS_LIBS)
```

File: app-text/atril/atril-1.26.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline app-text/atril/atril-1.26.2.ebuild:4]: 8 should work here.

File: app-text/atril/atril-1.26.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+MATE_LA_PUNT="yes"
+
+PYTHON_COMPAT=( python3_{10..11} )
```
Comment (juippis) [inline app-text/atril/atril-1.26.2.ebuild:8]:
```suggestion
PYTHON_COMPAT=( python3_{10..12} )
```
Should also work.

File: app-text/atril/atril-1.26.2.ebuild (hunk 1 above)
Comment (oz123) [reply app-text/atril/atril-1.26.2.ebuild:53]: Should I add a new flag `comics`?

File: app-text/atril/atril-1.26.2.ebuild (hunk 2 above)
Comment (oz123) [reply app-text/atril/atril-1.26.2.ebuild:4]: addressed.

File: app-text/atril/atril-1.26.2.ebuild (hunk 3 above)
Comment (oz123) [reply app-text/atril/atril-1.26.2.ebuild:8]: Fixed.

File: app-text/atril/atril-1.26.2.ebuild (hunk 1 above)
Comment (juippis) [reply app-text/atril/atril-1.26.2.ebuild:53]: Mmmm that or just add `app-arch/libarchive:=` dep unconditionally. I presume comics is always enabled, is there a reason to make it toggleable? You decide.

**Review by juippis — APPROVED** [review]

**Comment by oz123** [comment]
I didn't add `app-arch/libarchive:=` yet. Guess I'll do it in 1.28 which will be soon released.

**Comment by juippis** [comment]
06182fd0b32dc8e409d859a9f4ce48d721d2ff24

**Pull request closed without merging**